misk/web/tabs/README.md

```diff
@@ -1,4 +1,72 @@
-How to make a new tab
+How to make a new admin tab
 ---
 
-Coming soon...
\ No newline at end of file
+- For this document, the following tab properties are used in examples
+  - name: New Tab
```

Comment:
I wonder if you could use another example, just ’cause _Tab_ is sometimes a placeholder “new tab” and sometimes it’s not, like in `_tab/`. Maybe a fun example like dinosaurs?!

Reply:
New example incoming...🦎 🐊